app/src/main/java/org/simple/clinic/summary/medicalhistory/MedicalHistorySummaryView.kt

```diff
@@ -46,7 +46,7 @@ class MedicalHistorySummaryView(
   lateinit var activity: AppCompatActivity
 
   @Inject
-  lateinit var effectHandlerFactory: MedicalHistorySummaryEffectHandler.Factory
+  lateinit var effectHandlerFactory: MedicalHistorySummaryEffectHandler
```

Comment:
Can we rename this property since it's no longer a factory but the effect handler itself?